administrator/language/de-DE/plg_twofactorauth_totp.sys.ini

```diff
@@ -6,4 +6,4 @@
 ; Obsolete since 4.2.0 -- The entire file must be removed in Joomla 5.0
 
 PLG_TWOFACTORAUTH_TOTP="Zwei-Faktor-Authentifizierung – Google Authenticator"
-PLG_TWOFACTORAUTH_TOTP_XML_DESCRIPTION="Ermöglicht den Benutzern die Zwei-Faktor-Authentifizierung mittels Google Authenticator oder einem anderen Einmal-Passwort-Generator zu aktivieren. Um die Zwei-Faktor-Authentifizierung zu verwenden, muss im jeweiligen Benutzerprofil die Zwei-Faktor-Authentifizierung durch den Benutzer selbst konfiguriert werden."
+PLG_TWOFACTORAUTH_TOTP_XML_DESCRIPTION="Ermöglicht den Benutzern die Zwei-Faktor-Authentifizierung mittels <a href='https://en.wikipedia.org/wiki/Google_Authenticator' target='_blank' rel='noopener noreferrer'>Google Authenticator</a> oder einem anderen Einmal-Passwort-Generator wie <a href='https://freeotp.github.io/' target='_blank' rel='noopener noreferrer'>FreeOTP</a> zu aktivieren. Um die Zwei-Faktor-Authentifizierung zu verwenden, muss im jeweiligen Benutzerprofil die Zwei-Faktor-Authentifizierung durch den Benutzer selbst konfiguriert werden."
```

Comment:
```suggestion
PLG_TWOFACTORAUTH_TOTP_XML_DESCRIPTION="Ermöglicht den Benutzern die Zwei-Faktor-Authentifizierung mittels <a href='https://de.wikipedia.org/wiki/Google_Authenticator' target='_blank' rel='noopener noreferrer'>Google Authenticator</a> oder einem anderen Einmal-Passwort-Generator wie <a href='https://freeotp.github.io/' target='_blank' rel='noopener noreferrer'>FreeOTP</a> zu aktivieren. Um die Zwei-Faktor-Authentifizierung zu verwenden, muss im jeweiligen Benutzerprofil die Zwei-Faktor-Authentifizierung durch den Benutzer selbst konfiguriert werden."
```

Reply:
Und in dieser Datei müsste der selbe string auch angepasst werden: https://github.com/joomlagerman/joomla/blob/fd5a7be6470d8bbd7464b5be57370679ff3dd2c4/administrator/language/de-DE/plg_twofactorauth_totp.ini